public/about.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Document</title>
+      <link rel="stylesheet" href="../css/about.css">
+</head>
+<body>
+      <header></header>
+      <main>
+            <div id="aboutTitle">
+                  <h3>ABOUT</h3> 
+                  <h1>CARLA BEAUTY</h1>      
+            </div>
+            <div id="aboutBody">
+                  <div id="aboutContent">
+                        <div class="border-container">
+                              <img src="https://s3-alpha-sig.figma.com/img/53fe/3e37/cb900b356f370ec25b7589a78b78a87b?Expires=1732492800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=EHhhDwUZbEyx8PXoUVOHHlcZGtEG5BTwvLl75EVxH0Ti2c1ff8Uo8bLhRLp1sAfclXDoyo2f2VSuLju4wCJ8I2K-qeaQIGKx3Hz-1svtFVrTiGswsEhVNuUg63GXsS1e0NOot8qq99XvDXsqgOgIBsWkvPvMfu4M8xzq774RfUIwzsWWFOgJqwf7t-lxxlbblhYZulXfrv2LctAsbLAcWrAfiw8PmscLfTOdonkqxyOtjLA0jK6qiBZL~j8qWmcb3IbkdwvwonET32iGPRE2m3dqF9ZgdNN1wQPZ9ksiMjn90fquqRRPV3uVY01R-ysdn3uFMf~-8QmGhGWFsTe9fw__" alt="">
+                        </div>
+                        <div class="content">
+                              <h3>Hi, I’m Carla Beatriz,<br/>the Founder of Carla Beauty.</h3>
+                              <p>At Carla Beauty, our commitment lies in personalized service in enhancing natural beauty of individuals. With a focus on the client's style and skin type, we create customized makeup looks using high-quality products and staying updated on the latest trends. As a L'Oréal Certified professional makeup artist with over a decade of experience in Brazil and Canada, my goal is to provide clients with an unforgettable experience that leaves them feeling beautiful and confident.</p>
+                              <div id="btn"> 
+                                    <button>My Work</button>      
+                              </div>
```

Comment:
I suggest change these code like this. 
```suggestion
<button id="btn">My Work</button>
```
And in scss, you can ....
```suggestion
#btn {
      text-align: center;
      font-size: 18px;
      padding: 20px 32px;
}
```